src/pull_module/hf_pull_model_module.cpp

```diff
@@ -93,12 +93,12 @@ std::variant<ovms::Status, std::unique_ptr<Libgt2InitGuard>> createGuard() {
 
 Status HfPullModelModule::start(const ovms::Config& config) {
     state = ModuleState::STARTED_INITIALIZE;
-    SPDLOG_INFO("{} starting", HF_MODEL_PULL_MODULE_NAME);
+    SPDLOG_TRACE("{} starting", HF_MODEL_PULL_MODULE_NAME);
```

Comment:
Inconsistent with other modules

Reply:
Yes this is big change but those logs do not provide any crucial info for the user when pulling models, manipulating config or listing models.